fprettify/__init__.py

```diff
@@ -1549,6 +1549,9 @@ def get_arg_parser(args):
                             help="Paths to files to be formatted inplace. If no paths are given, stdin (-) is used by default. Path can be a directory if --recursive is used.", default=['-'])
         parser.add_argument('-r', '--recursive', action='store_true',
                             default=False, help="Recursively auto-format all Fortran files in subdirectories of specified path; recognized filename extensions: {}". format(", ".join(FORTRAN_EXTENSIONS)))
+        parser.add_argument('-e', '--exclude', action='append',
+                            default=[], type=str,
+                            help="File or directory patterns to be excluded when searching for Fortran filesx to format")
```

Comment:
filesx --> typo?

Reply:
Yes, that was a typo. Just pushed a correction.